packages/truffle-box/lib/utils/index.js

```diff
@@ -71,7 +63,7 @@ module.exports = {
       setUpSpinner.succeed();
     } catch (error) {
       setUpSpinner.fail();
-      throw new Error(error);
+      throw error;
```

Comment:
The removal of these `new Error(error)` is something I never thought to do :)  Cool

Reply:
Yeah, I've noticed there's a bunch of `new Error(err)` stuff thrown around in the codebase that messes with useful error reporting/output.